modules/dnn/test/test_layers.cpp

```diff
@@ -618,6 +618,51 @@ TEST(Layer_LSTM_Test_Accuracy_with_, HiddenParams)
     normAssert(h_t_reference, outputs[0]);
 }
 
+TEST(Layer_Slice_1d_Test, Accuracy){
+
+    LayerParams lp;
+    lp.type = "Slice";
+    lp.name = "sliceLayer";
+
+    int splits = 2;
+    int length = 4;
+    int batch_size = 0;
```

Comment:
Will this test be parametrized?

Reply:
Probably not, since parametrization complicates `output_ref` calculation for all parametrized cases. The main aim is 1D tests.

Reply:
LGTM?